src/multidim/arithmetic.jl

```diff
@@ -31,11 +31,8 @@ Base.size(X::IntervalBox{2,Float64}) = (2,)
 @inline broadcasted(f, x, y, Z::IntervalBox) = wrap(f.(x, y, Z.v))
 @inline broadcasted(f, x, Y::IntervalBox, z) = wrap(f.(x, Y.v, z))
 
-for op in (:+, :-, :∩, :∪, :⊆, :isinterior, :dot, :setdiff, :×)
+for op in (:+, :-, :∩, :∪, :⊆, :⊂, :⊃, :isinterior, :dot, :setdiff, :×)
```

Comment:
What about `⊇`?

Reply:
It just works (Julia magic); see the tests. ;-)